src/rust/ide/src/model/project.rs

```diff
@@ -60,9 +61,19 @@ pub trait API:Debug {
     }
 
     /// Generates full module's qualified name that includes the leading project name segment.
-    fn qualified_module_name(&self, path:&model::module::Path) -> crate::model::module::QualifiedName {
+    fn qualified_module_name
+    (&self, path:&model::module::Path) -> crate::model::module::QualifiedName {
         path.qualified_module_name(self.name().deref())
     }
+
+    /// Look up given id in the suggestion database and if it is a known method obtain a pointer to
+    /// it.
+    fn method_ptr_from_db_id
+    (&self, id:SuggestionId) -> FallibleResult<language_server::MethodPointer> {
+        let db    = self.suggestion_db();
+        let entry = db.lookup(id)?;
+        language_server::MethodPointer::try_from(entry.as_ref()).map_err(Into::into)
+    }
```

Comment:
Can this function be implemented on SuggestionDatabase instead of Project Model?